docs/micro-lc/core_plugins.md

```diff
@@ -35,13 +35,13 @@ the plugin **always** injects 3 properties called `eventBus`, `currentUser`, and
 - _type_: string;
 - _required_: `false`;
 - _description_: tag of the custom element to render.  
-  **Is mandatory only for type `element`**.
+  **Description is mandatory only for `element` type**.
```

Comment:
Why description? This property is called tag.

Reply:
Ok here i messed up, I'm sorry.
I suggest to write the following sentence and to bring it right below the name:

> **This property is mandatory  only for `element` type**.